compiler/ir/backend.js/src/org/jetbrains/kotlin/ir/backend/js/JsLoweringPhases.kt

```diff
@@ -660,6 +662,20 @@ private val blockDecomposerLoweringPhase = makeBodyLoweringPhase(
     prerequisite = setOf(typeOperatorLoweringPhase, suspendFunctionsLoweringPhase)
 )
 
+private val stringConcatenationLowering = makeBodyLoweringPhase(
+    ::StringConcatenationLowering,
+    name = "StringConcatenationLowering",
+    description = "Lowers String concatenation to a chain of String::plus",
+    prerequisite = setOf(blockDecomposerLoweringPhase)
```

Comment:
The `blockDecomposerLoweringPhase` step creates String concatenations, so it needs to run first